### PR TITLE
Alina

### PR DESCRIPTION
**Pull request by alinaincodeland**
Worked on home page.

### CODE REVIEW

**Review by Rami24t — APPROVED** [review]

File: client/src/components/contactForm.css [hunk 1]
```diff
@@ -1,53 +1,82 @@
...
+    height: fit-content;
+}
+
+
+.contact-form-section {
+    max-width: 1160px;
+    margin: auto;
+}
+
+.form-title {
```
Comment (Rami24t) [inline client/src/components/contactForm.css:14]: make this .contact-form-title

File: client/src/components/description.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  margin: auto;
+}
+
+.description-title {
+  font-size: 27px;
+  margin-top: 2rem;
+  margin-bottom: 1rem;
+}
+
+.stripy-image {
```
Comment (Rami24t) [inline client/src/components/description.css:42]: .description-stripy-image

File: client/src/components/hero.css [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    font-size: 56px;
+    max-width: 550px;
+}
+
+.hero-subtitle {
+    font-size:20px;
+    max-width: 480px;
+}
+
+.demo-button{
```
Comment (Rami24t) [inline client/src/components/hero.css:29]: .hero-demo-button

File: client/src/components/slider.scss [hunk 4]
```diff
@@ -0,0 +1,73 @@
+@mixin white-gradient {
+	background: linear-gradient(to right, rgba(255, 255, 255, 1) 0%, rgba(255, 255, 255, 0) 100%);
+}
+
+$animationSpeed: 40s;
+
+// Animation
+@keyframes scroll {
```
Comment (Rami24t) [inline client/src/components/slider.scss:8]: slider-scroll

File: client/src/components/slider.scss [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+}
+
+ .slider-subtitle {
+	font-size: 14px !important;
+		font-weight: 500 !important;
+		padding-left: 0px;
+		text-align: center;
+		color: rgba(44, 17, 56, 0.6);
+ }
+.slide img {
```
Comment (Rami24t) [inline client/src/components/slider.scss:69]: .slider slide img?

File: client/src/index.scss [hunk 6]
```diff
@@ -0,0 +1,98 @@
+@import url('https://fonts.googleapis.com/css2?family=Lora:wght@400;500;600;700&family=Oswald:wght@200;300;400;500;600;700&display=swap');
```
Comment (Rami24t) [inline client/src/index.scss:1]: we already have an index.css file.
did you take it into consideration when making this index.scss file?

File: client/src/reset.scss [hunk 7]
```diff
@@ -0,0 +1,329 @@
+/* Improve media defaults */
```
Comment (Rami24t) [inline client/src/reset.scss:1]: Are you sure you want to apply this reset to all the pages?

File: client/src/index.scss (hunk 6 above)
Comment (alinaincodeland) [reply client/src/index.scss:1]: I replaced the index.css with index.scss file :) we don't have both files.

File: client/src/reset.scss (hunk 7 above)
Comment (alinaincodeland) [reply client/src/reset.scss:1]: Yes, it covers potential compatibility issues across different browsers.

**Review by ju8ko — APPROVED** [review]

**Review by Rami24t — CHANGES_REQUESTED** [review]

**Review by Rami24t — APPROVED** [review]

**Pull request merged**